GPU/GLES/TextureCacheGLES.cpp

```diff
@@ -512,10 +508,8 @@ void TextureCacheGLES::BuildTexture(TexCacheEntry *const entry, bool replaceImag
 	}
 
 	// Always generate a texture name unless it's a framebuffer, we might need it if the texture is replaced later.
```

Comment:
This comment is no longer relevant.  It was true when this was above `entry->framebuffer` and was about how we were assigning a texture name to the entry so that replaceImages wouldn't crash later, even for framebuffers.  Now we don't do that.

-[Unknown]